db/migrate/20151222152212_remove_vm_discover_row_from_miq_event_definitions.rb

```diff
@@ -0,0 +1,11 @@
+class RemoveVmDiscoverRowFromMiqEventDefinitions < ActiveRecord::Migration
+  class MiqEventDefinition < ActiveRecord::Base
+    self.inheritance_column = :_type_disabled
+  end
+
+  def up
+    say_with_time("Remove event definition vm_discover") do
+      MiqEventDefinition.where(:name => 'vm_discover').delete_all
```

Comment:
@bzwei This needs to use `destroy_all` so dependent relationships are also removed.  In this case the `MiqEventDefinition` has a relationship to `MiqEventDefinitionSet` (through the `Relationship` model) that will not be properly removed if you use `delete_all`.